.coafile

```diff
@@ -1,7 +1,7 @@
 [all]
 bears = LineCountBear, FilenameBear
 files = **.py, **.yaml, **.rst
-file_naming_convention = snake
+file_naming_convention = camel
```

Comment:
I changed to camel because tests were failing for names like ```buildConfig-template.yaml```

Reply:
Python code should be snake_case, yaml files are CamelCase.